actions/actions.go

```diff
@@ -132,16 +132,19 @@ func (s *service) doWork(ctx context.Context) error {
 		iteration int
 	)
 
-	b := backoff.WithContext(backoff.WithMaxRetries(backoff.NewConstantBackOff(5*time.Second), 3), ctx)
-	errR := backoff.Retry(func() error {
-		iteration++
```

Comment:
if we are removing this, why we are printing it then?

Reply:
Good point, I didn't notice we already have "log-per-retry" here so we double-log now. Will fix.

Reply:
I think I implemented your goal, we now log an error on every failed attempt and it has the iteration and actual error observed - please check.